inspirehep/modules/submissions/serializers/schemas/author.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from marshmallow import Schema, fields, pre_dump, missing, post_load
+from inspire_schemas.builders.authors import AuthorBuilder
+from inspire_utils.record import get_value, get_values_for_schema
+
+
+class Author(Schema):
+    display_name = fields.Raw()
+    family_name = fields.Raw()
+    given_name = fields.Raw()
+    native_names = fields.Raw()
+    public_emails = fields.Raw()
+
+    status = fields.Raw()
+    arxiv_categories = fields.Raw()
+    websites = fields.Raw()
+    twitter = fields.Raw()
+    blog = fields.Raw()
+    linkedin = fields.Raw()
+
+    positions = fields.Raw()
+    project_membership = fields.Raw()
+    advisors = fields.Raw()
+
+    comments = fields.Raw()
+
+    acquisition_source = fields.Raw()
+
+    @pre_dump
+    def before_dump(self, data):
+        author_dict = {}
+
+        family_name, given_name = self.get_name_splitted(data)
+        author_dict.update({
```

Comment:
what's the difference between doing this and returning directly the argument to `update`? here it looks like you are building two `dict`s for no good reason.